tasks/summarize-zap-results/task.rb

```diff
@@ -11,8 +11,12 @@ def project_name
 end
 
 puts "Comparing last results to current results..."
-ZAPResultSetComparator.write_summary(last_run_dir, current_run_dir, output_dir, project_name)
+comparator = ZAPResultSetComparator.new(project_name, last_run_dir, current_run_dir)
+if comparator.no_change?
```

Comment:
It prevents it from posting in slack:
> when text is omitted, and...  text_file omitted, or present but file missing: > (no notification given)

From https://github.com/cloudfoundry-community/slack-notification-resource

I guess the behavior of that resource could change on us.